### PR TITLE
belinda-nextjs-refactor-410-add-belinda-designs

### PR DESCRIPTION
**Pull request by tinpham5614**
Related issue #410
Related PR #412 

This PR will follow up on the previous PR, it will switch category images/icons based on the current theme mode as demo screenshot.

### Latest update

**Dark mode**
image

**Light mode**
image

### CODE REVIEW

**Comment by Diego-Cano** [comment]
Hi! I reviewed the latest pull request on theme-based category images. The implementation looks great and really improves the user experience by switching images for light and dark modes. The use of useTheme() from MUI to dynamically switch images is a great idea. It’s awesome to see how the image paths are structured for both modes and conditionally set based on the current theme as the changes in the code reflect.

 It might be good idea to test thoroughly at some point to catch any potential edge cases. Also, adding a fallback mechanism for image paths to enhance robustness would be great too.

I found this useful link on working with SVGs and its benefits: 
[SVG Primer](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial). 
image

**Comment by tinpham5614** [comment]
@Diego-Cano 
Thank you for the insightful feedback.
I noticed that the page takes a bit longer to load with PNG images.
I will convert these PNG images to SVG.

**Review by Asfand00 — APPROVED** [review]
LGTM tin, nice work.

light mode:
light_page

dark mode:
dark_page

response is fast too.

File: components/CategoryImages.tsx
Comment (gitbiruk2010) [inline components/CategoryImages.tsx:1]: I have never implemented theme changes like dark and light. This is a great feature to add. the items look cool in both theme preferences. good job Tin.

**Review by BradleyCharles — APPROVED** [review]
Working like a charm. Looking great!

Screenshot 2024-05-30 194608
Screenshot 2024-05-30 194622

**Comment by tinpham5614** [comment]
Should I replace the top left logo with this Belinda text logo?
image

**Review by JoyZhang2023 — APPROVED** [review]
Work like a charm. Good job! One thing to mention is that I got an error message about module installation. I ran `npm install reaci-icons` to install the package and the error goes away and everything works perfectly. @tinpham5614 It seems main branch doesn't have react-icon package. Would you like to add it to this pull request? 
image
image
image

**Review by Seiyaroo — APPROVED** [review]
Gonna just echo the comments of great job. SVG is a nice catch! I think changing to Belindas Logo makes the most sense, and would you be able to implement @JoyZhang2023 suggestion @tinpham5614 ? Either way, great work as always!

**Comment by tinpham5614** [comment]
@JoyZhang2023 
Thank you for the feedback.
The react-icons package is in the main branch. You may need to run `npm install react-icons` once on the local machine.

**Review by intisarosman1 — APPROVED** [review]
Looks good! the images go really well with the color themes now.
Screenshot 2024-05-31 000425

Screenshot 2024-05-31 000411

**Pull request merged**